packages/forklift-console-plugin/src/modules/Plans/views/details/components/PlanWarningCondition.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Linkify from 'react-linkify';
+
+import { Alert, Text, TextContent, TextVariants } from '@patternfly/react-core';
+
+export const PlanWarningCondition: React.FC<{
+  type: string;
+  message: string;
+  suggestion: string;
+}> = ({ type, message, suggestion }) => {
+  const { t } = useTranslation();
+  return (
+    <Alert title={t('The plan migration might not work as expected - ') + type} variant="warning">
+      <TextContent className="forklift-providers-list-header__alert">
+        <Text component={TextVariants.p}>
+          <Linkify>{message || '-'}</Linkify>
```

Comment:
It is better to export '-' as a const and then consume it here; it is probably used in many places and then can be changed quickly if needed

Reply:
Sure, replaced on other occurrences as well